test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -52,10 +52,10 @@ export default class PossibleIssues extends Component {
         // fetch related issues with Github API
         let additionalRepo = "";
         if (buildName.includes('j9') || buildName.includes('ibm')) {
-            additionalRepo = "+repo:eclipse/openj9";
+            additionalRepo = "+repo:eclipse-openj9/openj9";
         }
-        const response = await fetch(`https://api.github.com/search/issues?q=${generalTestName}+state:open+repo:AdoptOpenJDK/openjdk-tests` +
-                    `+repo:AdoptOpenJDK/openjdk-infrastructure+repo:AdoptOpenJDK/openjdk-build+repo:adoptium/aqa-systemtest+repo:adoptium/TKG${additionalRepo}`, {
+        const response = await fetch(`https://api.github.com/search/issues?q=${generalTestName}+state:open+repo:adoptium/aqa-tests` +
+                    `+repo:adoptium/aqa-infrastructure+repo:adoptium/aqa-build+repo:adoptium/aqa-systemtest+repo:adoptium/TKG${additionalRepo}`, {
```

Comment:
https://github.com/AdoptOpenJDK/openjdk-infrastructure is not switched yet.

Reply:
Corrected